app/substrait_producer_function_compatibility.py

```diff
@@ -0,0 +1,125 @@
+from typing import List, Optional
+
+import ibis
+import pandas as pd
+import streamlit as st
+from ibis import _
+
+st.set_page_config(layout="wide")
+
+
+def support_matrix_df():
+    return (
+        ibis.read_csv(
+            "https://raw.githubusercontent.com/richtia/consumer-testing/producer_support_matrix/app/producer_results.csv"
```

Comment:
Update for the public repo (or access this locally so it can be updated in tests).  (And elsewhere.)

Reply:
doesn't exist in the public repo yet.  should i just go ahead and make the change now with the future location once this is merged? or should i wait to do it in another PR?

Reply:
What's the rationale for pointing this to a remote location?  If the path is local the format of the support matrix and the program to utilize it could be changed in one PR without breaking anything.

Reply:
hmmm...no particular reason actually...I'm okay with just using the relative path here too.  I'll go ahead and do that.